### PR TITLE
Improve contribution guide

### PR DESCRIPTION
**Pull request by marianeum**
Our contribution guide was lacking details - particularly around screenshot testing & docs screenshots. Added some more details & clarification to make it easier to understand.

+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/main/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] Component `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -85,26 +89,48 @@ To start the emulator and attach an SD card to it, run
...
 
 ```
 ./scripts/verify_screenshot_tests.sh
 ```
 
-#### Espresso tests
+If the check fails you either need to fix the issue if a change was unintended or record script above instead to update the screenshots.
+
+### Espresso tests
+If your component contains logic that can't be verified via screenshot tests you can use espresso to test the logic. These tests live in the `Backpack` or `backpack-compose` module, depending on the component.
```
Comment (anambl) [inline CONTRIBUTING.md:124]: tests that live in `Backpack` are for `View` components?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -85,26 +89,48 @@ To start the emulator and attach an SD card to it, run
 $ANDROID_HOME/tools/emulator -avd bpk-droid-avd -sdcard sd.img &
 ```
 
-After adding new snapshot tests, run
+#### Creating tests
+
+Snapshot tests live in the `app` module. For a new test class extend from `BpkSnapshotTest`.
+Each test function will result in a separate test function - try to add a test for each state of a component.
```
[the remark this answers is not in the document]
Comment (marianeum) [reply CONTRIBUTING.md:95]: yes that was friday brain 😂 let me fix that

File: CONTRIBUTING.md (hunk 1 above)
Comment (marianeum) [reply CONTRIBUTING.md:124]: i'll clarify that

**Review by anambl — APPROVED** [review]
LGTM!

**Pull request merged**